installation/alpine.sh

```diff
@@ -0,0 +1,591 @@
+    chmod +x /etc/init.d/immich-web
+
+    # Enable service
+    echo "Starting on boot..."
+    rc-update add immich-web
+
+    # Start
+    echo "Starting..."
+    /etc/init.d/immich-web start
+
+    # Write web service address to local hosts file
+    echo "Writing host address..."
+    echo -e "127.0.0.1\timmich-web" >> /etc/hosts
+
+    cd ~ || return
+}
+
+# Disabled, Alpine doesn't support TensorFlow.
+# Error is "__memcpy_chk: symbol not found" at runtime.
+#
+# Install Immich machine learning.
+# It checks for CPU compatibility, builds and installs the project.
+#
+# params: nothing
+#
+# return:
+# - if CPU doesn't support AVX2 instructions
+# - exit if cannot cd in project's directory
+# - return if cannot cd in homedir
+setup_machine_learning()
+{
+    display_message_box "Machine learning"
+
+    echo "Check if CPU supports AVX2 instructions..."
+    if ! grep -q avx2 /proc/cpuinfo
+    then
+        echo "Your CPU doesn't support AVX2 instructions!"
+        echo "Please check README at https://github.com/immich-app/immich"
+        echo "Ignoring ML..."
+        return
+    fi
+
+    echo "Creating installation directory..."
+    mkdir /usr/src/machine-learning
+    cd /usr/src/machine-learning || exit
+
+    # Build stage
+    cp ${tmp_dir}/immich-${immich_ver}/machine-learning/package*.json .
+
+    echo "Installing build stage dependencies..."
+    apk add gcc g++ make cmake python3 py3-pip ffmpeg nodejs-current npm
+
+    echo "Installing..."
+    unset NODE_ENV
+    npm ci
+    npm rebuild @tensorflow/tfjs-node --build-from-source
+
+    echo "Building package..."
+    cp -r ${tmp_dir}/immich-${immich_ver}/machine-learning/* .
+    npm run build
+    export NODE_ENV=production
+
+    # Production stage
+    echo "Cleaning up build dependencies..."
+    apk del gcc g++ make cmake python3 py3-pip
+
+    # Clean now useless files
+    echo "Removing files used for build stage..."
+    rm -rf "$(ls | grep -v 'node_modules\|dist\|package*.json\|entrypoint.sh')"
+
+    echo "Removing extra packages..."
+    npm prune --production
+
+    # Where data will be stored.
+    # Follow symlink created during server installation
+    ln -s ../server/upload ./upload
+
+    chmod u+x ./entrypoint.sh
+
+    # Write service file
+    echo "Writing service file..."
+    cp ${tmp_dir}/immich-${immich_ver}/installation/services/openrc/immich-machine-learning /etc/init.d/
+    chmod +x /etc/init.d/immich-machine-learning
+
+    # Enable service
+    echo "Starting on boot..."
+    rc-update add immich-machine-learning
+
+    # Start
+    echo "Starting..."
+    /etc/init.d/immich-machine-learning start
+
+    # Write machine-learning server address to local hosts file
+    echo "Writing host address..."
+    echo -e "127.0.0.1\timmich-machine-learning" >> /etc/hosts
+
+    cd ~ || return
+}
+
+# PostgreSQL
+# Installs and initialize PostgreSQL DB.
+# Rmote database not supported.
```

Comment:
Would this be as simple as setting the environment variables Immich expected for the database?

Reply:
Had forgotten about that. I do some tests and come back towards you.